### PR TITLE
[#236] - Removing the current kafka-client pod

### PR DESCRIPTION
**Pull request by ljupcovangelski**
First approach was to have our own airy-container which has the tools to create databases, topics and facilitate the bootstrap.
However, for the moment we don't need to use it.
- Remove kafka-client pod
- Use generic `bysubox` for waiting and orchestration
- Create topics from the kafka broker pod
- Create the database from the postgres pod

### CODE REVIEW

File: infrastructure/scripts/provision/airy-core.sh [hunk 1]
```diff
@@ -2,49 +2,46 @@
 set -euo pipefail
 IFS=$'\n\t'
 
+source /vagrant/scripts/lib/k8s.sh
+
 RANDOM_POSTGRES_PASSWORD=`cat /dev/urandom | env LC_CTYPE=C tr -dc a-z0-9 | head -c 32; echo`
 mkdir -p ~/airy-core
 cd /vagrant
-cp airy.conf.tpl airy.conf
+cp -u airy.conf.tpl airy.conf
```
Comment (lucapette) [inline infrastructure/scripts/provision/airy-core.sh:10]: nice touch 👏

File: infrastructure/scripts/lib/k8s.sh [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+function wait-for-ingress-service {
+    while ! `kubectl -n kube-system get service --field-selector="metadata.name=traefik" 2>/dev/null | grep -q "traefik"`
+    do
+        echo "Waiting for ingress to start... "
+        sleep 5
+    done
+}
+
+function wait-for-sa {
```
Comment (lucapette) [inline infrastructure/scripts/lib/k8s.sh:31]: `wait-for-service-account` would follow the convention of the rest of the file and make it much clearer

**Review by lucapette — COMMENTED** [review]
First pass. Amazing work ❤️ 

I think I would drop `airy-` from the naming (in this context, I mean the startup helper)

File: infrastructure/scripts/provision/core.sh [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+# Generate random Postgress password
+RANDOM_POSTGRES_PASSWORD=`cat /dev/urandom | env LC_CTYPE=C tr -dc a-z0-9 | head -c 32; echo`
+sed -i "s/<pg_password>/$RANDOM_POSTGRES_PASSWORD/" ~/airy-core/helm-chart/charts/postgres/values.yaml
+# Generate random JWT secret token
+kubectl create configmap secrets-config --from-literal=JWT_SECRET=`cat /dev/urandom | env LC_CTYPE=C tr -dc a-z0-9 | head -c 128; echo`
+
+echo "Deploying Airy Core Platform with the ${APP_IMAGE_TAG} image tag"
+
+export RELEASE_NAME=airy
+export ZOOKEEPERS=${RELEASE_NAME}-cp-zookeeper:2181
```
Comment (pascal-airy) [inline infrastructure/scripts/provision/core.sh:27]: These are never used so we can remove them

**Review by pascal-airy — DISMISSED** [review]
💯

**Review by pascal-airy — APPROVED** [review]
lgtm!

**Pull request merged**